UIKitDZ/UIKitDZ/StopWatchModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  StopWatch.swift
+//  UIKitDZ
+//
+//  Created by Max Inedom on 7/9/21.
+//
+
+import Foundation
+
+final class StopWatchModel {
+    
+    //MARK: - Private Properties
+    private var startTime: NSDate?
+    private var elapsedTime: TimeInterval {
+        if let startTime = self.startTime {
+            return -startTime.timeIntervalSinceNow
+        } else {
+            return 0
+        }
+    }
+    
+    //MARK: - Public Properties
+    public var isRunning: Bool {
+        return startTime != nil
+    }
+    
+    public var elapsedTimeAsString: String {
+        return String(format:"%02d:%02d", Int(elapsedTime / 60), Int(elapsedTime.truncatingRemainder(dividingBy: 60)))
+    }
+    
+    public func start() {
```

Comment:
```suggestion
    func startTimer() {
```
public не пишем никогда

Reply:
изменил